core/io/resource_loader.cpp

```diff
@@ -808,6 +808,12 @@ String ResourceLoader::_path_remap(const String &p_path, bool *r_translation_rem
 		if (r_translation_remapped) {
 			*r_translation_remapped = true;
 		}
+
+		// Fallback to p_path if new_path does not exist.
+		if (!FileAccess::exists(new_path)) {
+			WARN_PRINT_ED(vformat("Translation Remap %s does not exist. Falling back to %s", new_path, p_path));
```

Comment:
Some style suggestions:
```suggestion
			WARN_PRINT(vformat("Translation remap '%s' does not exist. Falling back to '%s'.", new_path, p_path));
```

Also it seems we have no code anywhere in the codebase using the `_ED` variant of the warning/error macros so that seems to indicate that either we need to start using them consistently if they're useful, or remove them if they're not. It's outside the scope of this PR so I'd suggest just using the regular `WARN_PRINT`.